requirements-transitive.txt

```diff
@@ -36,6 +37,7 @@ kombu==4.6.11
 lazy-object-proxy==1.4.3
 MarkupSafe==1.1.1
 mccabe==0.6.1
+moto==3.1.4
```

Comment:
same for this one, but maybe we put test dependencies here?

Reply:
Yeah, lets keep test dependency here